service/common/src/main/java/org/apache/polaris/service/catalog/IcebergCatalogAdapter.java

```diff
@@ -459,6 +461,25 @@ public Response listViews(
         .build();
   }
 
+  @Override
+  public Response loadCredentials(
+      String prefix,
+      String namespace,
+      String table,
+      RealmContext realmContext,
+      SecurityContext securityContext) {
+    Namespace ns = decodeNamespace(namespace);
+    TableIdentifier tableIdentifier = TableIdentifier.of(ns, RESTUtil.decodeString(table));
+    LoadTableResponse loadTableResponse =
+        newHandlerWrapper(realmContext, securityContext, prefix)
+            .loadTableWithAccessDelegation(tableIdentifier, "ALL");
```

Comment:
Iceberg REST API spec appears to require the snapshots parameter to be lower case, i.e. `all`: https://github.com/apache/polaris/blob/main/spec/iceberg-rest-catalog-open-api.yaml#L948